src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -53,11 +53,17 @@ class Gemini(session: SparkSession) {
 
 case class RepoFile(repo: String, file: String, sha: String)
 
+case class DuplicateBlobHash(sha: String, count: Long) {
+  override def toString(): String = s"$sha ($count duplicates)"
+}
+
 object Gemini {
   val defaultCassandraHost: String = "127.0.0.1"
   val defaultCassandraPort: String = "9042"
   val defaultSchemaFile: String = "src/main/resources/schema.cql"
 
+  val cql = "SELECT blob_hash, repo, file_path, COUNT(*) as count FROM hashes.blob_hash_files GROUP BY blob_hash"
```

Comment:
do you think `duplicatesCountCql` or something like this would be better?

Reply:
why do you need to select `repo` and `file_path` here if you are grouping anyway?

Reply:
actually it's weird. I don't see support of GROUP BY in cassandra. How does it work?

Reply:
@smacker it's rather new https://github.com/scylladb/scylla/issues/2206
and that is why this PR has 2 implementations: with and without it.

Reply:
So I can't use it right now with scylla? :(

Reply:
In Cassandra since more than 1y ago. There is not in Scylla :(

Reply:
Yes. Let's make a no-group-by a default behaviour for now